src/components/Boundary/Boundary.stories.tsx

```diff
@@ -2,23 +2,26 @@ import {storiesOf} from '@storybook/react-native'
 import React from 'react'
 import {Button, StyleSheet, Text, View} from 'react-native'
 import {ActivityIndicator} from 'react-native-paper'
-import {QueryClient, QueryClientProvider, useQuery} from 'react-query'
+import {useQuery} from 'react-query'
 
+import {QueryProvider} from '../../../storybook/decorators/query'
```

Comment:
export from `index.ts`
```suggestion
import {QueryProvider} from '../../../storybook/decorators'
```

Reply:
done thanks